android/AndroidManifest.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+          package="com.creature.android"
```

Comment:
You can change this package name if you so desire, just do a full project search to see where else string `com.creature.android` needs to be changed.